pkg/apply/applier.go

```diff
@@ -46,8 +47,8 @@ func NewApplier(factory util.Factory, ioStreams genericclioptions.IOStreams) *Ap
 	}
 	a.previousInventoriesFunc = a.PruneOptions.GetPreviousInventoryObjects
 	a.infoHelperFactoryFunc = a.infoHelperFactory
-	a.InventoryFactoryFunc = prune.WrapInventoryObj
-	a.PruneOptions.InventoryFactoryFunc = prune.WrapInventoryObj
+	a.InventoryFactoryFunc = inventory.WrapInventoryObj
+	a.PruneOptions.InventoryFactoryFunc = inventory.WrapInventoryObj
```

Comment:
Is there any way we can avoid doing this? Not necessarily in this PR. But it makes it impossible to hide PruneOptions behind an interface.

Reply:
Yes -- in a future PR.